example/integration_tests/bazel5/WORKSPACE.bazel

```diff
@@ -13,11 +13,9 @@ register_graalvm_toolchains()
 
 load("@rules_graalvm//graalvm:repositories.bzl", "graalvm_repository")
 
-rules_graalvm_repositories()
-
 graalvm_repository(
     name = "graalvm",
-    distribution = "oracle",
+    distribution = "ce",
```

Comment:
examples and tests now default to `ce` instead of `oracle`, for safer copy-paste as starter projects.